lambda_powertools/prometheus.py

```diff
@@ -0,0 +1,78 @@
+import os
+import json
+import sys
+
+modulename = 'prometheus_client'
+if modulename in sys.modules:
+    raise Exception("prometheus_client already imported, lambda_powertools.prometheus must be imported before")
+
+os.environ['PROMETHEUS_DISABLE_CREATED_SERIES'] = 'True'
+import prometheus_client
+from prometheus_client import CollectorRegistry
+
+prometheus_client.REGISTRY.unregister(prometheus_client.GC_COLLECTOR)
+prometheus_client.REGISTRY.unregister(prometheus_client.PLATFORM_COLLECTOR)
+prometheus_client.REGISTRY.unregister(prometheus_client.PROCESS_COLLECTOR)
+
+
+def reset():
+    for collector in list(prometheus_client.REGISTRY._collector_to_names.keys()):
+        if "_value" in dir(collector):
+            # case Counter without labels
+            collector._value.set(0)
+        elif "_metrics" in dir(collector):
+            for metric in collector._metrics.values():
+                if "_value" in dir(metric):
+                    # case Counter with labels
+                    metric._value.set(0)
+                elif "_buckets" in dir(metric) and "_sum" in dir(metric):
+                    # case Histogram with labels
+                    metric._sum.set(0)
+                    for bucket in metric._buckets:
+                        bucket.set(0)
+        elif "_buckets" in dir(collector) and "_sum" in dir(collector):
+            # case Histogram without labels
+            collector._sum.set(0)
+            for bucket in collector._buckets:
+                bucket.set(0)
+
+
+def filter_metrics(m):
+    # Other metric types are not supported so far
+    if not m._type == "counter" and not m._type == "histogram":
```

Comment:
Why not:
`if m._type != "counter" and m._type != "histogram":`
or:
`if m._type not in ("counter", "histogram"):`